server/middleware/checkWhetherToShowServiceOnboardingBanner.test.ts

```diff
@@ -1,5 +1,6 @@
 import { Request, Response } from 'express'
 import checkWhetherToShowServiceOnboardingBanner from './checkWhetherToShowServiceOnboardingBanner'
+import ApplicationRole from '../enums/applicationRole'
```

Comment:
The unit tests in this file have had to change, because the impl has changed (next file down)